### PR TITLE
test: more cases for webSocketURL

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [X] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

more test cases -

- test port on `auto`
- test port on `80`
- client.webSocketURL.port on `0` option
- `host` on ipv4 

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

**Comment by snitin315** [comment]
It looks like for port `80` case we need `root` privilege on Github actions.

File: test/e2e/web-socket-server-and-url.test.js [hunk 1]
```diff
@@ -454,6 +454,144 @@ for (const webSocketServerType of webSocketServerTypes) {
...
+  describe('should work with the "port" option is 80', () => {
+    beforeAll((done) => {
+      const options = {
+        webSocketServer: webSocketServerType,
+        port: 80,
+        host: '0.0.0.0',
+        client: {
+          webSocketURL: {
+            port: port2,
+          },
```
Comment (alexander-akait) [inline test/e2e/web-socket-server-and-url.test.js:500]: Can we add all tests without `webSocketURL`?, otherwise tests do not have sense

Comment (snitin315) [reply test/e2e/web-socket-server-and-url.test.js:500]: I will update.

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**